demo/index.html

```diff
@@ -25,12 +25,10 @@
     import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
     import './simple-expand-collapse.js';
 
-    const $_documentContainer = document.createElement('template');
-
-    $_documentContainer.innerHTML =
-        `<custom-style><style is="custom-style" include="demo-pages-shared-styles"></style></custom-style>`;
-
-    document.body.appendChild($_documentContainer.content);
+    import {html} from '@polymer/polymer/lib/utils/html-tag.js';
+    const template = html
+    `<custom-style><style is="custom-style" include="demo-pages-shared-styles"></style></custom-style>`;
```

Comment:
You can put the backtick on the line above so you can get syntax highlighting

Reply:
```js

Reply:
```js
html`
<custom-style>...</custom-style>`;
```

Reply:
I already made this change in multiple elements - don't think it's necessary to go through all these repos again just for this change

Reply:
Sure, but you can still make the change here